doc/ssl/SSL_CTX_set_default_passwd_cb.pod

```diff
@@ -12,8 +12,13 @@ callback for encrypted PEM file handling
 
  void SSL_CTX_set_default_passwd_cb(SSL_CTX *ctx, pem_password_cb *cb);
  void SSL_CTX_set_default_passwd_cb_userdata(SSL_CTX *ctx, void *u);
+ pem_password_cb *SSL_CTX_get_default_passwd_cb(SSL_CTX *ctx);
+ void *SSL_CTX_get_default_passwd_cb_userdata(SSL_CTX *ctx);
+
  void SSL_set_default_passwd_cb(SSL *s, pem_password_cb *cb);
  void SSL_set_default_passwd_cb_userdata(SSL *s, void *u);
+ pem_password_cb *SSL_get_default_passwd_cb(SSL *s);
+ void *SSL_get_default_passwd_cb_userdata(SSL *s);
```

Comment:
You need to add the above functions to the NAME section above too. Also add a HISTORY section at the bottom stating that they were added in 1.1.0  (e.g. see the HISTORY section in SSL_CTX_config.pod)